src/sortStudents.ts

```diff
@@ -1,16 +1,52 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averagegrade',
```

Comment:
just a small thing , it would be more convenient and readable in this way :
```suggestion
  AverageGrade = 'averageGrade',
```